juriscraper/opinions/united_states/federal_special/tax.py

```diff
@@ -30,6 +31,22 @@ def __init__(self, *args, **kwargs):
         }
 
     def _download(self, request_dict={}):
+        """Download from api
+
+        The tax court switches between blue and green deploys so we need to
+        check which one is current before we continue
+
+        :param request_dict:
+        :return: None
```

Comment:
Don't forget these, please.

Reply:
yes.

Reply:
Sounds good.  I'll probably just delete the param completely as I don't think it's needed .